OrbitLinuxTracing/UprobesReturnAddressManager.h

```diff
@@ -59,6 +61,38 @@ class UprobesReturnAddressManager {
     }
   }
 
+  void PatchCallchain(pid_t tid, uint64_t* callchain, uint64_t nr,
+                      unwindstack::Maps* maps) {
+    if (!tid_uprobes_stacks_.contains(tid)) {
+      return;
+    }
+
+    auto& tid_uprobes_stack = tid_uprobes_stacks_.at(tid);
+    CHECK(!tid_uprobes_stack.empty());
+
+    // TODO(kuebler): What about tail-call optimization, where two uretprobes
+    //  hijacked an address at the same stack pointer.
+    auto uprobes_it = tid_uprobes_stack.rbegin();
+    for (uint64_t i = 0; i < nr; i++) {
+      uint64_t ip = callchain[i];
+      unwindstack::MapInfo* map_info = maps->Find(ip);
+
+      // Only patch Broken IPs
```

Comment:
Maybe remove this comment and add a comment for the function describing briefly what is being done (as all the comments related to this class are for "stack patching" but not for "callchain patching").